com.unity.ml-agents/Tests/Editor/Actuators/ActuatorManagerTests.cs

```diff
@@ -64,6 +64,18 @@ public void TestEnsureBufferDiscrete()
             Assert.IsTrue(7 == manager.StoredActions.DiscreteActions.Length);
         }
 
+        [Test]
+        public void TestAllowMixedActions()
```

Comment:
A version of this (that checked for failure) was removed in the hybrid action PR. Adding it back to make sure we can do it.